### ISSUE TITLE
npm package contains ~220K of undo/backup files

### ISSUE THREAD

**Issue by exogen** [felixge/node-stream-cache#3]
Ideally this would be republished with a modern version of npm, which uses a better default `.npmignore`. Or, add an `.npmignore` manually.

<img width="846" alt="screen shot 2016-09-21 at 12 43 56 pm" src="https://cloud.githubusercontent.com/assets/53522/18726386/175e6f28-7ff9-11e6-9431-18b9c05ac6ab.png">

**Comment by SpaceK33z** [comment]
Please use [`files`](https://docs.npmjs.com/files/package.json#files) instead, it's better to whitelist what files to include.

**Comment by exogen** [comment]
@SpaceK33z I remember switching off `files` and back to `.npmignore` on some projects before due to issues.

Check out another project with this issue, [gauge](https://github.com/iarna/gauge/issues/63) for example – they [do have a correct `files` config](https://github.com/iarna/gauge/blob/master/package.json#L42-L62) and yet the temporary files `CHANGELOG.md~` and `README.md~` are still there, most likely due to this npm rule:

> - README (and its variants)
> - CHANGELOG (and its variants)

(variants probably means: anything beginning with `README` or `CHANGELOG`)

**Comment by SpaceK33z** [comment]
True, but that's a feature. And it is actually quite nice. These pages can be very handy when debugging in node_modules.

**Comment by exogen** [comment]
@SpaceK33z I agree, but it's including the undo/backup files `README.md~` and `CHANGELOG.md~` as well.

Actually it looks like `.npmignore` suffers from this problem as well, even if you have `*~` it will keep around `README*~` and `CHANGELOG*~` 😞
